pkg/asset/targets/targets.go

```diff
@@ -60,6 +60,14 @@ var (
 		&cluster.Metadata{},
 	}
 
+	// SingleNodeIgnitionConfig is the bootstrap-in-place ignition-config targeted assets.
+	SingleNodeIgnitionConfig = []asset.WritableAsset{
+		&kubeconfig.AdminClient{},
+		&password.KubeadminPassword{},
+		&bootstrap.Bootstrap{BootstrapInPlace: true},
```

Comment:
This is not valid. You need to have a separate asset type. You can have that separate asset type compose `bootstrap.Bootstrap`.

The asset store differentiates between asset by their types. So if you use the same type for both bootstrap ignition assets, the asset store will not be able to tell them apart. The consequence of this is that if the user were to run `openshift-install create ignition-configs` followed by `openshift-install create single-node-ignition-config`, then the resulting ignition config would not have the bootstrap-in-place bits.